src/cli-validator/utils/processConfiguration.js

```diff
@@ -10,6 +10,8 @@ const defaultConfig = require('../../.defaultsForValidator');
 // global objects
 const readFile = util.promisify(fs.readFile);
 const defaultObject = defaultConfig.defaults;
+// Clear spectral rules from defaultObject
```

Comment:
I think this makes sense but I was confused at first - maybe just add a bit more info in the comment like 
```suggestion
// Clear spectral rules from defaultObject, only use the explicit values in the validaterc file
// the default values for spectral come from another file
```